dags/multipod/legacy_tests/gpt1-like.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python3
```

Comment:
This file is copied directly from our old unit tests repository. The only changes are from `code-style.sh`

Reply:
Curious what's the resource? I thought your team's tests are all in Jsonnet

Reply:
Also, do we want to add Copyright header? as a OS standard

Reply:
Good catch! I'll add that.

Our internal tests are defined in jsonnet, but they call into unit test scripts.